### ISSUE TITLE
Fatal error: Uncaught Error: Class 'Doctrine\Instantiator\Instantiator' not found

### ISSUE THREAD

**Issue by Th3Ov3r** [doctrine/couchdb-odm#146]
please help me, it may be happening.
here my file boostrap.php
`<?php
$couchPath = "lib";
require_once $couchPath . "/vendor/doctrine-common/lib/Doctrine/Common/ClassLoader.php";

$loader = new \Doctrine\Common\ClassLoader("Doctrine\Common", $couchPath . "/vendor/doctrine-common/lib");
$loader->register();

$loader = new \Doctrine\Common\ClassLoader("Doctrine\ODM\CouchDB", $couchPath);
$loader->register();

$loader = new \Doctrine\Common\ClassLoader("Doctrine\CouchDB", $couchPath);
$loader->register();

$loader = new \Doctrine\Common\ClassLoader("Symfony", $couchPath . "/vendor");
$loader->register();

$annotationNs = 'Doctrine\\ODM\\CouchDB\\Mapping\\Annotations';
Doctrine\Common\Annotations\AnnotationRegistry::registerAutoloadNamespace($annotationNs, $couchPath);

$databaseName = "couchdb_odm";
$documentPaths = array("MyApp\Documents");
$httpClient = new \Doctrine\CouchDB\HTTP\SocketClient();
$dbClient = new Doctrine\CouchDB\CouchDBClient($httpClient, $databaseName);

$config = new \Doctrine\ODM\CouchDB\Configuration();
$metadataDriver = $config->newDefaultAnnotationDriver($documentPaths);

$config->setProxyDir(__DIR__ . "/proxies");
$config->setMetadataDriverImpl($metadataDriver);
$config->setLuceneHandlerName('_fti');

$dm = new \Doctrine\ODM\CouchDB\DocumentManager($dbClient, $config);

use MyApp\Document\BlogPost;

require_once 'MyApp/Documents/BlogPost.php';

$blogPost = new BlogPost();
$blogPost->setHeadline("Hello World!");
$blogPost->setText("This is a blog post going to be saved into CouchDB");

$dm->persist($blogPost);
$dm->flush();`

**Comment by Th3Ov3r** [comment]
Here my file BlogPost.php
`<?php
namespace MyApp\Document;

/** @Document */
class BlogPost
{
    /** @Id */
    private $id;
    /** @Field(type="string") */
    private $headline;
    /** @Field(type="string") */
    private $text;

    public function setHeadline($headline)
    {
        $this->headline = $headline;
    }

    public function getHeadline()
    {
        return $this->headline;
    }

    public function setText($text)
    {
        $this->text = $text;
    }

    public function getText()
    {
        return $this->text;
    }

}`

**Comment by Ocramius** [comment]
The doctrine class loader should no longer be used, as composer completely replaces it: are your examples coming from the docs? If so, then we'll likely need to upgrade or deprecate that section.

**Comment by Th3Ov3r** [comment]
Yes, the examples take them from doctrine couchdb documentation.
What is the correct way to download and load the libs, help me please.

**Comment by mikeSimonson** [comment]
On the command line you can ``composer require doctrine/couchdb-odm`` and then you just need to require_once the vendor/autoload.php file.

Can you point us where you found the outdated documentation that show the use of the doctrine  classLoader ?

**Comment by Th3Ov3r** [comment]
http://docs.doctrine-project.org/projects/doctrine-couchdb/reference/configuration.html#class-loading

**Comment by mikeSimonson** [comment]
Thanks for the docs link.

The error that you are seeing from composer is weird.
It looks like there is an invisible character because the error mention the right package.
Also you shouldn't require doctrine/couchdb-odm in what looks like a git clone from doctrine/couchdb-odm but in your project where you want to use it.

**Comment by Th3Ov3r** [comment]
Yes, very strange. Try other libs, such as, for example,

composer require "doctrine/mongodb-odm"

If it worked well.

**Comment by Th3Ov3r** [comment]
try another pc. I keep throwing error. But this time the
exception is different. What could be happening?

**Comment by mikeSimonson** [comment]
Sorry but I have no idea what's going on, but it's not a couchdb-odm bug.
